consensus/istanbul/core/roundchange.go

```diff
@@ -30,11 +30,11 @@ import (
 
 // sendNextRoundChange sends the ROUND CHANGE message with current round + 1
 func (c *core) sendNextRoundChange(loc string) {
-	cv := c.currentView()
-	if c.backend.NodeType() == common.CONSENSUSNODE {
-		logger.Warn("[RC] sendNextRoundChange happened", "where", loc)
+	if c.backend.NodeType() != common.CONSENSUSNODE {
+		return
 	}
-	c.sendRoundChange(new(big.Int).Add(cv.Round, common.Big1))
+	logger.Warn("[RC] sendNextRoundChange happened", "where", loc)
+	c.sendRoundChange(new(big.Int).Add(c.currentView().Round, common.Big1))
```

Comment:
So, has it broadcasted round change messages although the node is not CN? Shouldn't we check the unnecessary consensus message communication between the PNs/ENs?

Reply:
@jeongkyun-oh The broadcasting and handling logic is working in PN/EN, but PN/EN doesn't gossip consensus messages to others. The gossiping function checks whether the node is in the committee or not. 
Anyway, we should check and refine PN/EN's behaviors in the consensus logic sooner or later.

Reply:
I see. Thanks a lot!